utils/factories/user.ts

```diff
@@ -0,0 +1,21 @@
+import { Factory } from "fishery";
+import { User } from "@prisma/client";
+import prisma from "../../prisma/client";
+
+export default Factory.define<User>(({ sequence, onCreate }) => {
+  onCreate(user => {
+    return prisma.user.create({ data: user });
+  });
+
+  return {
+    id: sequence,
+    discordId: sequence.toString().padStart(17, '0'),
+    discordAvatar: "???",
```

Comment:
We can also leave this null as `???` can never be the value for this. Then special case can be used for the N-ZAP user of `f809176af93132c3db5f0a5019e96339` that's their avatar value that I never change it so it can be used for testing like this.